utils/cache-domains/files/cache-domains.init

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh /etc/rc.common
+
+START=24
+SERVICE_NAME=cache-domains
+CACHE_DOMAINS_DIR="/var/${SERVICE_NAME}"
+CACHE_DOMAINS_SRC="https://api.github.com/repos/uklans/cache-domains/tarball/master"
+CONFIG_FILE="/etc/${SERVICE_NAME}.json"
+
+start() {
+	mkdir -p ${CACHE_DOMAINS_DIR}
+	rm -fr ${CACHE_DOMAINS_DIR}/*
+
+	wget -qO - ${CACHE_DOMAINS_SRC} | tar -xzC ${CACHE_DOMAINS_DIR}
```

Comment:
A check of exit status and return immediately can be convenient.

Past experience tells me wget with tls has its fair chance of not working out of the box.  It can be uclient-fetch lacking ssl variant of ustream libraries.  It can be absence or expiration of ca-certificates to pass tls verification.

Reply:
good call! 
Turns out I was getting false positives in my testing before and perchance had the certificates.
I added `ca-bundle` and checked for an error/return.

also, it seems that `wget` with `wolfss` l doesn't work with https so I removed that variant.
I suspect it's the same issue as https://forum.openwrt.org/t/should-uclient-fetch-work-with-wolfssl/38163/9